references/detection/train_tensorflow.py

```diff
@@ -359,9 +374,9 @@ def parse_args():
         formatter_class=argparse.ArgumentDefaultsHelpFormatter,
     )
 
-    parser.add_argument("train_path", type=str, help="path to training data folder")
-    parser.add_argument("val_path", type=str, help="path to validation data folder")
     parser.add_argument("arch", type=str, help="text-detection model to train")
+    parser.add_argument("--train_path", type=str, help="path to training data folder")
+    parser.add_argument("--val_path", type=str, help="path to validation data folder")
```

Comment:
I changed that so it's the same options than `recognition`. I'll do the same for PyTorch in another PR.

Reply:
See comment :)